lib/pbio/platform/ev3/platform.c

```diff
@@ -489,6 +490,17 @@ void ev3_panic_handler(int except_type, ev3_panic_ctx *except_data) {
     panic_puts("\r\nSPSR: 0x");
     panic_putu32(except_data->spsr);
 
+    uint32_t r;
+    __asm__ volatile ("mrc p15, 0, %0, c5, c0, 0" : "=r" (r));
```

Comment:
These need comment to explain what the assembly code does. (Or even better, use `cp15.h` functions instead of inline assembly).